assets/queries/cloudFormation/aws/s3_bucket_access_to_any_principal/query.rego

```diff
@@ -19,25 +19,22 @@ CxPolicy[result] {
 
 	common_lib.any_principal(policyStatements[_])
 
-	publicAccessBlockConfiguration := resourceBucket.Properties.PublicAccessBlockConfiguration
-
-	targets := {"RestrictPublicBuckets", "IgnorePublicAcls"}
-	publicAccessBlockConfiguration[targets[t]] == false
-
 	result := {
 		"documentId": input.document[indexBucket].id,
 		"resourceType": resourceBucket.Type,
 		"resourceName": cf_lib.get_resource_name(resourceBucket, nameBucket),
-		"searchKey": sprintf("Resources.%s.Properties.PublicAccessBlockConfiguration.%s", [nameBucket, targets[t]]),
+		"searchKey": sprintf("Resources.%s.Properties.PolicyDocument", [nameBucket]),
```

Comment:
We should improve the `searchKey` and `searchLine`.  AWS::S3::Bucket resource does not have the field 'PolicyDocument'